src/state/middleware/logging.ts

```diff
@@ -1,5 +1,7 @@
 import { createLogger } from 'redux-logger'
+import { ReduxState } from 'state/reducer'
 
-export const logging = createLogger({
-  predicate: getState => getState()?.preferences?.featureFlags?.ReduxLogging
+export const loggingMiddleware = createLogger({
+  predicate: (getState: () => ReduxState) => getState()?.preferences?.featureFlags?.ReduxLogging,
+  diff: true
```

Comment:
is there a way to configure this to show the diff as a full object, rather than a json diff style spec?

unfortunately when we add like 6000 assets they're being serialized individually, to the point where the log spew isn't adding value.

Reply:
Hey @0xdef1cafe sorry it's taken a while to get back to this one. I'm afraid there's no options to tweak the diff output for `redux-logger` to get you an object you can inspect in the console. I guess this goes back to what I was saying [here](https://github.com/shapeshift/web/pull/1251#issuecomment-1076388857) in that if you want a redux logger you can mould to be what you want, it would be better to role you own rather than rely on a 3rd party for the implementation.

If you would like, for a small bounty, I could roll the requirements of this PR https://github.com/shapeshift/web/pull/1251 and the requirement to have a diff as an object, in to a custom redux middleware for the project that would do both?

Reply:
@afenton90 sure - provide a quick estimate in terms of hours where we can get a similarly formatted output to what we have currently, with a collapsible diff option, and we'll gladly bounty that for you

Reply:
Great stuff! Estimate for the logging middleware would be 3 hours. If you bounty that up, I'll get on with that in a new PR 👍